central/scannerdefinitions/handler/handler.go

```diff
@@ -493,17 +509,26 @@ func openFromArchive(archiveFile string, fileName string) (*os.File, error) {
 	return tmpFile, nil
 }
 
-func (h *httpHandler) getV4Files(w http.ResponseWriter, r *http.Request, updaterKey, fileName string) {
-	log.Infof("Fetching repository mapping file: %s", fileName)
+func (h *httpHandler) getV4Files(w http.ResponseWriter, r *http.Request, t updaterType, updaterKey, fileName string) {
+	log.Debugf("Fetching scanner V4 file: %s", fileName)
 
 	var err error
 	var f *vulDefFile
 
-	switch updaterKey {
-	case mappingUpdaterKey:
-		f, err = h.openMostRecentFile(mappingUpdaterKey, fileName)
+	switch t {
+	case mappingUpdaterType:
+		f, err = h.openMostRecentV4File(t, mappingUpdaterKey, fileName)
+	case vulnerabilityUpdaterType:
+		if version.GetVersionKind(updaterKey) == version.NightlyKind {
+			// get dev for nightly at this moment
+			updaterKey = "dev"
```

Comment:
We have nightlies for release branches, don't we? Shouldn't we get the appropriate version for it?

Reply:
If we are going to publish bundles for nightly versions, that will be too much for the bucket. Also the customers won't use the nightly bundles. So I set to dev bundle for now.

Reply:
I agree, I think using `dev` for nightlies makes sense